src/app/address/controllers/addressConfirm.test.js

```diff
@@ -35,8 +35,16 @@ describe("Address confirmation controller", () => {
 
     req.sessionModel.set("addresses", addresses);
 
+    // factor in the address might have building name or number or both
```

Comment:
I feel this is a little odd, isn't your controller suppose to handle this why is the test doing this logic all over, maybe I don't understand the test

Reply:
A bit of context:

We're using the HMPO form wizard to handle our forms, sessions and user journeys. 
As a result, there's a predefined lifecycle when entering each page on the form. One of those steps is the local's step which initialises any local variables specific for that page and view, the locals function here that we're testing is testing the locals are formatted in the correct way to be displayed on the page to the user.

 
 For more info on the HMPO form wizard take a look here: 
 https://github.com/HMPO/hmpo-form-wizard#get-lifecycle